Kernel/Bus/USB/xHCI/DeviceTreexHCIController.cpp

```diff
@@ -11,7 +11,7 @@
 #include <Kernel/Firmware/DeviceTree/Driver.h>
 #include <Kernel/Firmware/DeviceTree/Management.h>
 
-namespace Kernel::USB {
+namespace Kernel::USB::xHCI {
```

Comment:
I moved all xHCI code to this namespace so the new headers don't put xHCI stuff into the Kernel::USB namespace.